crates/cli/src/tasks/rust.rs

```diff
@@ -20,12 +21,15 @@ fn cargo_cmd(subcommand: &str, build_debug: bool, args: &[&str]) -> duct::Expres
 }
 
 pub(crate) fn build_rust(project_path: &Path, skip_clippy: bool, build_debug: bool) -> anyhow::Result<PathBuf> {
-    // Make sure that we have the wasm target installed (ok to run if its already installed)
-    if let Err(err) = cmd!("rustup", "target", "add", "wasm32-unknown-unknown").run() {
-        println!(
-            "Warning: Failed to install wasm32-unknown-unknown target: {}. Is `rustup` installed?",
-            err
-        );
+    if !has_wasm32_target() {
+        if has_rust_up() {
+            // Make sure that we have the wasm target installed (ok to run if its already installed)
```

Comment:
at least part of this comment is no longer relevant (since if we run this code, we have already checked that the wasm32 target is _not_ installed)